### DISCUSSION TITLE
Grove Vision AI Module V2 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1229] [Q&A]
# Grove Vision AI Module V2 | Seeed Studio Wiki

Getting Started with Grove Vision AI V2.

https://wiki.seeedstudio.com/Grove-vision-ai-v2-camera-supported/

**Comment by neoscell** [comment]
How can we use the raspberry pi V2 or V3 cameras? I just tried a V3 and it was green. What do I need to do or are these not supported?

**Reply by Love4yzp** [reply]
Due to the firmware, using the Pi cameras, it's normal to have the green screen, using OV5647 could be better.

**Comment by tanmay10dolkar** [comment]
Are there any better camera sensors, other than OV5647, that are supported by the Grove Vision Al Module V2?

**Comment by Santhoshkumar27** [comment]
I saw that Grove Vision AI V2 now supports Raspberry Pi Camera Modules (V1, V2) after the February 2024 update. Can you please confirm if the Raspberry Pi Camera Module V3 (12MP) is also supported with the Grove Vision AI V2? If not, is there a roadmap or driver update planned for V3 compatibility?

**Reply by ReKST-117** [reply]
Hi, sorry we don't have the driver update for Raspberry Pi Camera Module V3 yet:(

**Comment by eerke2021** [comment]
if whenever I try upload a model I get stuck at 1 or 2% despite both .exe drivers above, 3 browsers & 4 cables do you think bootloader is corrupted?

**Reply by 12-Twelve-12** [reply]
Hi,The issue you’re experiencing seems to be more complex. Could you please contact our technical support team at sensecap@seeed.cc for further assistance? We will help you diagnose the problem in detail.

**Comment by inxnik-design** [comment]
Hi Seeed team,
	•	Is there a deep sleep mode so that the  Grove Vision AI V2 can spend long periods (weeks) waiting between captures?
	•	Is the current consumption in deep sleep measured and documented for the entire board, not just the AI chip?
	•	I couldn’t find a “real” deep sleep: when idle, all my similar boards (including ESP ones) draw at least ~70 mA, which means a 5000 mAh battery doesn’t last a week. Have you observed or tested this too — or is there a way to reduce it?

**Reply by Jasionf** [reply]
Our current firmware does not support deep sleep configuration. Deep sleep can be implemented with the ESP32 by pulling the reset pin low.

**Comment by imnotwannafire** [comment]
Is there any way to enter boot mode without usb cable? I Only provide power through 3V3 pin

**Reply by grilliiiiii** [reply]
> Is there any way to enter boot mode without usb cable? I Only provide power through 3V3 pin

Entering boot mode is required for proper firmware flashing, and this normally relies on the USB interface. 
May I know more about your application scenario?

**Comment by romankiss** [comment]
Hi, I have successful running for couple months eight boards of the Grove Vision AI V2 connected to the ESP32-S3 via the UART AT Commands. There are using different models such as Face detection, Gesture detection, Sports Car detection and Person detections, everything is working well. 

Past couple days i have a trouble with flashing a model such as Sports Car Detection and also with my own custom models stored in my account (rkiss). The flashing is starting alright, but around 70% is either hanging or terminating with an error message for unplug cable. Note, that the flashing model like Gesture or Face is sometime perfectly included resetting a device and calling Invoke. I have tried many time to flash a model Sports Car Detection, but always unsuccessful.

It looks like, something has been changed in the process of the deploying, it is very fuzzy for flashing the model, now i have tried again to flash a simple model such as Gesture Detection, the flashing started normal, i can see progressing percentage, but after 59% is flashing stopped for few seconds and then run faster percentage until 100% and exited with an error.

I have tried to use another board, but the issue is the same such as unsuccessful flashing a model. Note, that this issue i didn't have it a month ago, always flashing any model to the board was successful.

**Reply by limengdu** [reply]
The behavior you're describing — flashing hanging around 70% or terminating with an error — can sometimes be caused by server-side caching issues. I'd suggest trying a hard refresh of the webpage (Ctrl+Shift+R on Windows, or Cmd+Shift+R on Mac) to clear your browser cache, then attempt the model flashing again to see if the issue resolves.

**Comment by Tomo8080** [comment]
I plan to use Grove Vision AI Module V2 in conjunction with Edge Impulse. Unfortunately, the latest firmware from [seeed-grove-vision-ai-module-v2.zip](https://cdn.edgeimpulse.com/firmware/seeed-grove-vision-ai-module-v2.zip) does not include the "flash_mac.command" script as mentioned in your setup instructions.  The files included in the extracted zip file are:
xmodem folder
firmware.img 
README.txt

Is there another way I can flash the Edge Impulse firmware onto the board?

**Reply by Lukilyy** [reply]
Hello, regarding your question. You can use the CLI tool provided by Edge Impulse for flashing. For detailed flashing steps and command examples, you can refer to the instructions in the "Flash using Edge Impulse CLI tools" section of the HimaxWiseEyePlus repository:( https://github.com/HimaxWiseEyePlus/Seeed_Grove_Vision_AI_Module_V2?tab=readme-ov-file#flash-using-edge-impulse-cli-tools) If you encounter any problems during use, you can directly raise your questions in the warehouse.
